pandoc-slave/pandoc-workflow/pandoc-workflow.pri

```diff
@@ -0,0 +1,7 @@
+HEADERS += \
```

Comment:
I suggest to rename "pandoc-workflow.pri" to "pandoc-slave.pri". As current name is not suitable for this module.